src/pages/favorites/Favorites.js

```diff
@@ -0,0 +1,31 @@
+import React, {useEffect} from 'react';
+import '../../styles/pages/favorites.css';
+import Top from "./Top";
+import Contents from "./Contents";
+import {getFolderList} from "../../utils/api";
+
+function Favorites() {
+  const [data, setData] = React.useState(null);
+
+  const handleLoadFolderList = async () => {
+    let result;
+    try {
+      result = await getFolderList();
+      setData(result.folder);
+    } catch (error) {
+      console.log(error);
+    }
+  }
```

Comment:
```suggestion
    try {
      const folderResult = await getFolderList();
      setData(folderResult.folder);
    } catch (error) {
      console.log(error);
    }
  }
```
위처럼 const로 선언한 값을 사용해도 괜찮을것 같습니다!
그리고 이 함수의 이름을 그냥 getFolderList라고 짓는건 어떨까요? 저는 나름의 함수명 규칙이 있는데요.
이벤트를 다루는 이벤트에 대한 함수는 handle동작명, 그 외에 함수는 해당 동작만 다뤄요 그래서 이부분은 handle은 빼도 함수동작이 예측이 가능할것 같네요!